catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -208,6 +208,19 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
 			if (Utils.checkIsNetworkAvailableAndShowErrorMessage(WebViewActivity.this)) {
 				ToastUtil.showError(getBaseContext(), R.string.error_unknown_error);
 			}
+
+			if (errorCode == ERROR_CONNECT
+					|| errorCode == ERROR_FILE_NOT_FOUND
+					|| errorCode == ERROR_HOST_LOOKUP
+					|| errorCode == ERROR_TIMEOUT
+					|| errorCode == ERROR_UNKNOWN) {
```

Comment:
```suggestion
					|| errorCode == ERROR_UNKNOWN),
					|| errorCode == ERROR_PROXY_AUTHENTICATION) {
```
adding **ERROR_PROXY_AUTHENTICATION** should fix the problem which occurs on BrowserStack devices.
with this changes i have tested it on:
- Galaxy S9 Android 10 (real device)
- Pixel 4 Android 11 (Android Studio)
- Pixel 3 Android 9 (BrowserStack)
- GalaxyTab S5e (BrowserStack)
![image](https://user-images.githubusercontent.com/42325361/129341101-058608cb-d47d-4da5-bb25-394bddf76545.png)